app/pages/event1/member/RefundOfContributionsPage.scala

```diff
@@ -31,4 +33,11 @@ case object RefundOfContributionsPage extends QuestionPage[RefundOfContributions
   override def route(waypoints: Waypoints): Call = {
     routes.RefundOfContributionsController.onPageLoad(waypoints)
   }
+
+  override protected def nextPageNormalMode(waypoints: Waypoints, answers: UserAnswers): Page = {
+    answers.get(RefundOfContributionsPage) match {
+      case Some(_) => PaymentValueAndDatePage
```

Comment:
afaik, this page has radio buttons. if selection is mandatory, we can remove again matching and forwarding directly PaymentValueAndDatePage

Reply:
Resolved in next commit